lib/tax_tribunal/case.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 's3'
+
+module TaxTribunal
+  class Case
+    class CaseNotFound < StandardError; end
+    include TaxTribunal::S3
+    extend TaxTribunal::S3
```

Comment:
Might be worth using `module_function` in `TaxTribunal::S3` to avoid having to both include and extend this module.